prepare/templates/regression/bounded/pair.py

```diff
@@ -0,0 +1,24 @@
+from src.unitxt.catalog import add_to_catalog
+from src.unitxt.templates import OutputQuantizingTemplate, TemplatesList
+
+add_to_catalog(
+    OutputQuantizingTemplate(
+        input_format="""
+                   Given this text: '{text1}' and this text: '{text2}', on a scale of {min_value} to {max_value}, what is the {type_of_value} of this texts?
+                """.strip(),
+        output_format="{value}",
+        quantum=0.2,
+    ),
+    "templates.regression.bounded.pair.simple",
+    overwrite=True,
+)
+
+add_to_catalog(
+    TemplatesList(
+        [
+            "templates.regression.bounded.pair.simple",
+        ]
+    ),
+    "templates.regression.bounded.pair.all",
```

Comment:
I think it should be more clear:

templates.regression.bounded.pair -> templates.regression.two_texts

Is there a need for an unbounded regression between text?  
.